### PR TITLE
TF compile fix

### PR DESCRIPTION
**Pull request by SergeyTsimfer**
Fix compilation issues for multiple models

### CODE REVIEW

File: batchflow/tests/tf_models_compile_test.py [hunk 1]
```diff
@@ -1,57 +1,45 @@
...
 
 MODELS_SEG = [
     LinkNet,
     UNet,
-    VNet,  # fails
-    FCN32, FCN16, FCN8,
-    DenseNetFC56, DenseNetFC67, DenseNetFC103,
-    RefineNet, # fails
+    VNet,
+    FCN8, FCN16, FCN32,
```
Comment (roman-kh) [inline batchflow/tests/tf_models_compile_test.py]: You can remove these as they will hardly ever be used

File: batchflow/tests/tf_models_compile_test.py [hunk 2]
```diff
@@ -1,57 +1,45 @@
...
-    DeepLabXS, DeepLabX8, DeepLabX16
+    DeepLabXS, DeepLabX8, # DeepLabX16
 ]
 
 MODELS_CLF = [
     VGG16, VGG19, VGG7,
-    ResNet18, ResNet34, ResNet50, ResNet101, ResNet152,
-    ResNeXt18, ResNeXt34,  # ResNeXt50, ResNeXt101, ResNeXt152, # too heavy ?
-    # Inception_v1, Inception_v3, Inception_v4, InceptionResNet_v2,  # heavy fail
+    ResNet18, ResNet34, # ResNet50, ResNet101, ResNet152,
```
Comment (a-arefina) [inline batchflow/tests/tf_models_compile_test.py]: I think we need here an explanation, why some models are commented out

File: batchflow/tests/tf_models_compile_test.py [hunk 3]
```diff
@@ -1,58 +1,59 @@
...
+
+MODELS_CLF = [
+    VGG7, VGG16, VGG19,
+    ResNet18, ResNet34, # ResNet50, ResNet101, ResNet152,
+    ResNeXt18, ResNeXt34, # ResNeXt50, ResNeXt101, ResNeXt152,
+    Inception_v1,
+    # InceptionResNet_v2, Inception_v3, Inception_v4, # fail
+    SqueezeNet,
+    MobileNet, MobileNet_v2, MobileNet_v3, MobileNet_v3_small,
+    # DenseNet121,  # DenseNet169, DenseNet201, DenseNet264,
```
Comment (a-arefina) [inline batchflow/tests/tf_models_compile_test.py]: I think we should uncomment at least one model of DenseNet family

Comment (SergeyTsimfer) [reply batchflow/tests/tf_models_compile_test.py]: There is one: `DenseNetFC`

Comment (a-arefina) [reply batchflow/tests/tf_models_compile_test.py]: But they have different base classes.

File: batchflow/tests/tf_models_compile_test.py [hunk 4]
```diff
@@ -65,15 +56,28 @@ def base_config_clf():
...
 def test_clf(base_config_clf, model):
     """ Test models for classification """
     _ = model(base_config_clf)
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize('model', MODELS_SEG)
+def test_seg(base_config_segment, model):
+    """ Test models for segmentation """
+    _ = model(base_config_segment)
```
Comment (roman-kh) [inline batchflow/tests/tf_models_compile_test.py]: Fix

**Pull request merged**